src/scss/cdb-components/forms/checkbox.scss

```diff
@@ -15,11 +24,23 @@
 @import '../../cdb-variables/colors';
 
 .CDB-Checkbox {
+  width: $baseSize * 2;
```

Comment:
Properties should be ordered position, width, height, opacity, vertical-align, z-index